targets/lxde

```diff
@@ -11,6 +11,11 @@ CHROOTBIN='crouton-noroot startlxde'
 ### Append to prepare.sh:
 install lxde -- dictionaries-common chromium-browser network-manager wicd
 
+# xenial needs a few more packages.
+if release -ge xenial; then
+    install lxde-common lxsession-logout
```

Comment:
I think older Ubuntu releases pull in a reasonable set of packages. Not sure about Debian but I think it should be fine too.